classes/class-admin.php

```diff
@@ -18,6 +18,26 @@
  */
 class Admin {
 
+	/**
+	 * Used to check if it's a single site, not multisite.
+	 */
+	const WP_STREAM_SINGLE_SITE = 'single';
+
+	/**
+	 * Used to check if it's a multisite with the plugin network enabled.
+	 */
+	const WP_STREAM_MULTI_NETWORK = 'multisite-network';
+
+	/**
+	 * Used to check if it's a multisite with the plugin not network enabled.
+	 */
+	const WP_STREAM_MULTI_NOT_NETWORK = 'multisite-not-network';
```

Comment:
<!-- ideas that popped up from reviewing -->
**💡 thought:** Two things, nitpicks really:
1. I feel like this belongs to the `Plugin` class and not `Admin` (alternatively, this could be part of some kind of a `Utility` class). What's your take?
2. I guess it's a matter of preference but I wonder what do you think about having a set of methods for checking the installation type instead of those enum-like string constants? Something like `Admin::is_single_site()`, `Admin::is_multisite_network_activated()` and `Admin::is_multisite_not_network_activated()`.
It would follow the WordPress convention like `is_network_admin()` and make the conditions easier to read (potentially). But then they couldn't be used in a `switch` statement later on and a regular `if ... else` would have to be used.

Reply:
I agree with @delawski, but also have one comment about the constants naming:

I'm not sure what's the naming convention in this plugin, but addition of the `WP_STREAM_` prefix seems to not be necessary here (these constants are scoped in `Admin` class).

Reply:
> I feel like this belongs to the Plugin class and not Admin (alternatively, this could be part of some kind of a Utility class). What's your take?
Yes! I agree. I'll put it in the `Plugin` class for now and if it feels out of place there, move to a `Utility` class.

> I guess it's a matter of preference but I wonder what do you think about having a set of methods for checking the installation type instead of those enum-like string constants? Something like Admin::is_single_site(), Admin::is_multisite_network_activated() and Admin::is_multisite_not_network_activated().

Yes, I like this also, it makes sense to me and would make development easier.

Reply:
@bartoszgadomski Yes! You're right, that prefix isn't necessary and also not in line with the naming condition! I'll fix those.

Reply:
Commit : https://github.com/xwp/stream/pull/1543/commits/a6d2d36f9b95aaf0a0357e90051b25066b1ce4d3

If I missed doing something in that, it's accidental so please tell me